from staze import Error


class CellError(Error): pass
